SmartDeviceLink/SDLManager.h

```diff
@@ -84,6 +84,11 @@ typedef void (^SDLManagerReadyBlock)(BOOL success, NSError *_Nullable error);
  */
 @property (strong, nonatomic, readonly, nullable) SDLRegisterAppInterfaceResponse *registerResponse;
 
+/**
+ *  The auth token, if received. This should be used to log into a user account.
```

Comment:
Add "Primarily used for cloud apps with companion app stores."

Reply:
Fixed. Added